x/mongo/driver/topology/sdam_spec_test.go

```diff
@@ -457,6 +458,32 @@ func findServerInTopology(topo description.Topology, addr address.Address) (desc
 	return description.Server{}, false
 }
 
+func assertLogicalSessionTimeoutMinutesEqual(t *testing.T, expected *uint32, actual *uint32) {
+	t.Helper()
+
+	// If they pointers are equal, then do nothing.
+	if internal.CompareUint32Ptr(expected, actual) == 0 {
+		return
+	}
+
+	// Otherwise, include the pointer values in the failure message.
+	var expectedValue, actualValue interface{}
+	if expected != nil {
+		expectedValue = *expected
+	} else {
+		expectedValue = nil
+	}
+
+	if actual != nil {
+		actualValue = *actual
+	} else {
+		actualValue = nil
+	}
```

Comment:
Assigning `nil` is redundant because the zero value for `interface{}` is already nil.

```suggestion
	if actual != nil {
		actualValue = *actual
	}
```